netmaster/master/network.go

```diff
@@ -163,6 +164,57 @@ func deleteDockNet(networkName string) error {
 	return nil
 }
 
+func detachServiceContainer(networkName, tenantName string) error {
```

Comment:
yep, agree we need to think this through.

> Current thinking for skydns is to let schedulers(E.g. Swarm) run it anywhere in the cluster where resources are available

ok, I think for now we should use tcp socket here instead to be able to make use of swarm.

> ACI-gateway might have slightly different requirements around locality with active-netmaster.

hmm, do you have specific requirement in mind? Regarding `locality with active-netmaster` could we make use of REST endpoints (behind a VIP) to solve that?